libs/accounts/recovery-phone/src/lib/sms.manager.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Inject, Injectable } from '@nestjs/common';
+import { Twilio } from 'twilio';
+import { SmsManagerConfig } from './sms.manger.config';
+import { TwilioProvider } from './twilio.provider';
+
+@Injectable()
+export class SmsManager {
+  constructor(
+    @Inject(TwilioProvider) private readonly client: Twilio,
+    private readonly config: SmsManagerConfig
+  ) {}
+
+  public async sendSMS({ to, body }: { to: string; body: string }) {
+    return await this.client.messages.create({
```

Comment:
Does this need to be more complex? 
Do we want to put checks on to or body? 
Do we need to check and handle the result?